src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -85,10 +101,10 @@
     public final String user;
     public final String passwd;
     public final AuthProvider authProvider;
-    public final int throttle;
-    public final int interDcThrottle;
-    public final int entireSSTableThrottle;
-    public final int entireSSTableInterDcThrottle;
+    public final double throttleMegabits;
+    public final double interDcThrottleMegabits;
```

Comment:
@yifan-c , @frankgh considering our later discussions and the ticket that @yifan-c pointed to earlier today, probably we should find a way to workaround this change too? WDYT?

Reply:
I feel less worried about someone reading the field values... I can only image the use cases that use the option name constants to invoke the tool, or construct the option programmatically. The change does not affect those use cases.